packages/template-retail-react-app/app/utils/test-utils.js

```diff
@@ -132,10 +132,10 @@ export const TestProviders = ({
             return site.alias === siteAlias || site.id === appConfig['site']
```

Comment:
The `appConfig['site']` does not exist in the config anymore.

Reply:
Done. Good catch. I did a small refactor to re-use `getSiteByReference` in `TestProviders`